.github/workflows/wheels.yaml

```diff
@@ -23,29 +23,19 @@ jobs:
       - uses: actions/setup-python@v2
         name: Install Python
         with:
-          python-version: '3.9'
+          python-version: '3.10'
 
       - uses: actions/checkout@v2
       - name: Install cibuildwheel
-        run: python -m pip install cibuildwheel==2.2.2
-
-      - name: Install dependencies and yt_astro
-        shell: bash
-        env:
-          dependencies: "full"
-          LDFLAGS: "-static-libstdc++"
-        run: python -m pip install .
```

Comment:
reason for removing this: it is _not_ part of the normal cibuildwheel workflow, and was just inherited from yt's.
Furthermore, building the package in place creates shared objects (.so) files, which were the real reason why musllinux wheels were not building correctly and excluded